python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -219,7 +219,7 @@ def get_schema(cls):
               dataPath:
                 description: Path to the butler data repository.
```

Comment:
Maybe mention this is gen3?

Reply:
In general, we're trying not to do this anywhere - once Gen3 goes (which will be quite soon now, really) then Gen3 will be the only butler in town. I know this has merged, but just a general not to try not to do this if you can help it.